requirements.txt

```diff
@@ -31,3 +30,4 @@ twilio==3.4.1
 wsgiref==0.1.2
 python-dateutil==2.1
 django-auth-ldap==1.2.6
+sqlalchemy==1.1.5
```

Comment:
Since this is only used in the tests, shouldn't this be in `requirements-dev.txt`?